src/test/java/seedu/address/logic/parser/EditCommandParserTest.java

```diff
@@ -55,10 +61,13 @@
 import seedu.address.testutil.EditPersonDescriptorBuilder;
 
 public class EditCommandParserTest {
-
+    private static final String NAME_EMPTY = " " + PREFIX_NAME; // '&' not allowed in names
+    private static final String PROFILE_EMPTY = " " + PREFIX_PROFILE; // '^' not allowed in profile
+    private static final String PHONE_EMPTY = " " + PREFIX_PHONE; // 'a' not allowed in phones
+    private static final String EMAIL_EMPTY = " " + PREFIX_EMAIL; // missing '@' symbol
```

Comment:
I believe the comments don't match what the code is doing? Correct me if I'm wrong.

Reply:
You got it, I forgot to delete